### PR TITLE
fix(dashboards): Added tag loader to issue widgets query search

### PR DESCRIPTION
**Pull request by edwardgou-sentry**
fixes some missing tag values from issue widget query search:
image

### CODE REVIEW

**Review by narsaynorath — APPROVED** [review]
+1, looks good to me! Ran it locally and saw differences between prod and this branch.

File: static/app/components/dashboards/issueWidgetQueriesForm.tsx [hunk 1]
```diff
@@ -50,6 +54,19 @@ class IssueWidgetQueriesForm extends React.Component<Props, State> {
...
+    const {organization, selection} = this.props;
+    const orgId = organization.slug;
+    const projectIds = selection.projects.map(id => id.toString());
+    const endpointParams = {
+      start: getUtcDateString(selection.datetime.start),
+      end: getUtcDateString(selection.datetime.end),
+      statsPeriod: selection.datetime.period,
+    };
+
+    return fetchTagValues(this.props.api, orgId, key, search, projectIds, endpointParams);
```
Comment (shruthilayaj) [inline static/app/components/dashboards/issueWidgetQueriesForm.tsx:67]: nit: retrieve `api` when destructuring other props in line 58

File: tests/js/spec/components/dashboards/issueWidgetQueriesForm.spec.tsx [hunk 2]
```diff
@@ -6,6 +6,7 @@ import {generateFieldOptions} from 'sentry/views/eventsV2/utils';
 
 describe('IssueWidgetQueriesForm', function () {
   const organization = TestStubs.Organization();
+  const api = new MockApiClient();
```
Comment (shruthilayaj) [inline tests/js/spec/components/dashboards/issueWidgetQueriesForm.spec.tsx:9]: should we add a mock for get tags and add a test expecting it to be called?

**Review by shruthilayaj — APPROVED** [review]
LGTM!

File: static/app/components/dashboards/issueWidgetQueriesForm.tsx (hunk 1 above)
Comment (edwardgou-sentry) [reply static/app/components/dashboards/issueWidgetQueriesForm.tsx]: Fixed!

File: tests/js/spec/components/dashboards/issueWidgetQueriesForm.spec.tsx (hunk 2 above)
Comment (edwardgou-sentry) [reply tests/js/spec/components/dashboards/issueWidgetQueriesForm.spec.tsx:12]: 👍  added mock and test!

**Pull request merged**